src/main/scala/io/iohk/praos/domain/ledger/ConsensusResolver.scala

```diff
@@ -1,9 +1,26 @@
-package io.iohk.praos.domain
+package io.iohk.praos.domain.ledger
```

Comment:
Minor detail: Maybe it would be better to move this to `package io.iohk.praos.ledger`?

Reply:
sure!